addons/pkg/constants/constants.go

```diff
@@ -113,6 +113,9 @@ const (
 	// DiscoveryCacheInvalidateInterval is the interval for invalidating cache
 	DiscoveryCacheInvalidateInterval = time.Minute * 10
 
+	// PackageNameLabel is the label on the cloned objects namely Secrets and Providers by "TanzuClusterBootstrap" Reconciler to indicate the package name
+	PackageNameLabel = "tkg.tanzu.vmware.com/package-name"
```

Comment:
Question: Why `ClusterNameLabel` is in types, while `PackageNameLabel ` is in constants?

Reply:
I wanted to add it but since it was pre-existing, thought of re-using it.
Good idea would be to move it in constants and let other places use from here. What do you think @blc1996

Reply:
Can we move `PackageNameLabel ` to types?

Reply:
yes

Reply:
Fixed